nixos/tests/all-tests.nix

```diff
@@ -21,298 +21,298 @@ let
-  google-oslogin = handleTest ./google-oslogin {};
-  grafana = handleTest ./grafana.nix {};
-  graphite = handleTest ./graphite.nix {};
-  graylog = handleTest ./graylog.nix {};
-  gvisor = handleTest ./gvisor.nix {};
-  hadoop.hdfs = handleTestOn [ "x86_64-linux" ] ./hadoop/hdfs.nix {};
-  hadoop.yarn = handleTestOn [ "x86_64-linux" ] ./hadoop/yarn.nix {};
-  handbrake = handleTestOn ["x86_64-linux"] ./handbrake.nix {};
-  haproxy = handleTest ./haproxy.nix {};
+  #installed-tests = pkgs.recurseIntoAttrs (handleTest ./installed-tests {});
+  #gocd-agent = handleTest ./gocd-agent.nix {};
+  #gocd-server = handleTest ./gocd-server.nix {};
+  #google-oslogin = handleTest ./google-oslogin {};
+  #grafana = handleTest ./grafana.nix {};
+  #graphite = handleTest ./graphite.nix {};
+  #graylog = handleTest ./graylog.nix {};
+  #gvisor = handleTest ./gvisor.nix {};
+  #hadoop.hdfs = handleTestOn [ "x86_64-linux" ] ./hadoop/hdfs.nix {};
+  #hadoop.yarn = handleTestOn [ "x86_64-linux" ] ./hadoop/yarn.nix {};
+  #handbrake = handleTestOn ["x86_64-linux"] ./handbrake.nix {};
+  #haproxy = handleTest ./haproxy.nix {};
   hardened = handleTest ./hardened.nix {};
   hibernate = handleTest ./hibernate.nix {};
-  hitch = handleTest ./hitch {};
-  hocker-fetchdocker = handleTest ./hocker-fetchdocker {};
-  home-assistant = handleTest ./home-assistant.nix {};
-  hound = handleTest ./hound.nix {};
-  hydra = handleTest ./hydra {};
+  #hitch = handleTest ./hitch {};
+  #hocker-fetchdocker = handleTest ./hocker-fetchdocker {};
+  #home-assistant = handleTest ./home-assistant.nix {};
+  #hound = handleTest ./hound.nix {};
+  #hydra = handleTest ./hydra {};
   i3wm = handleTest ./i3wm.nix {};
-  icingaweb2 = handleTest ./icingaweb2.nix {};
-  iftop = handleTest ./iftop.nix {};
-  ihatemoney = handleTest ./ihatemoney.nix {};
-  incron = handleTest ./incron.nix {};
-  influxdb = handleTest ./influxdb.nix {};
+  #icingaweb2 = handleTest ./icingaweb2.nix {};
+  #iftop = handleTest ./iftop.nix {};
+  #ihatemoney = handleTest ./ihatemoney.nix {};
+  #incron = handleTest ./incron.nix {};
+  #influxdb = handleTest ./influxdb.nix {};
   initrd-network-ssh = handleTest ./initrd-network-ssh {};
   initrdNetwork = handleTest ./initrd-network.nix {};
   installer = handleTest ./installer.nix {};
   ipv6 = handleTest ./ipv6.nix {};
-  jackett = handleTest ./jackett.nix {};
-  jellyfin = handleTest ./jellyfin.nix {};
-  jenkins = handleTest ./jenkins.nix {};
-  kafka = handleTest ./kafka.nix {};
-  keepalived = handleTest ./keepalived.nix {};
-  kerberos = handleTest ./kerberos/default.nix {};
-  kernel-latest = handleTest ./kernel-latest.nix {};
+  #jackett = handleTest ./jackett.nix {};
+  #jellyfin = handleTest ./jellyfin.nix {};
+  #jenkins = handleTest ./jenkins.nix {};
+  #kafka = handleTest ./kafka.nix {};
+  #keepalived = handleTest ./keepalived.nix {};
+  #kerberos = handleTest ./kerberos/default.nix {};
+  #kernel-latest = handleTest ./kernel-latest.nix {};
```

Comment:
we have the latest in the releases so I think that's not the best omission either, testing being omitted is fine though.